suite-common/suite-utils/src/device.ts

```diff
@@ -88,11 +90,51 @@ export const deviceNeedsAttention = (deviceStatus: ReturnType<typeof getStatus>)
     }
 };
 
+export const shouldDisplayInitialWarningIcon = (deviceStatus: ConnectedDeviceStatus | null) => {
+    if (!deviceStatus) {
+        return false;
+    }
+
+    switch (deviceStatus) {
+        case 'bootloader':
+        case 'initialize':
+            return false;
+        default:
+            return true;
+    }
+};
+
+export const getDeviceStatusWarningVariant = (
+    deviceStatus: ReturnType<typeof getStatus>,
+): 'warning' | 'info' => {
+    switch (deviceStatus) {
+        case 'bootloader':
+        case 'initialize':
+            return 'info';
+        default:
+            return 'warning';
+    }
+};
+
+export const getDeviceResolveStatusCTAMessage = (deviceStatus: ReturnType<typeof getStatus>) => {
+    switch (deviceStatus) {
+        case 'bootloader':
+        case 'initialize':
+            return 'TR_SELECT_DEVICE_SHORT';
+        default:
+            return 'TR_SOLVE_ISSUE';
+    }
+};
+
 export const getDeviceNeedsAttentionMessage = (deviceStatus: ReturnType<typeof getStatus>) => {
+    // V bootloaderr:  bootloader
+    // wipedd: initialize
+    // without FW: bootloader
```

Comment:
Is it intented to have it here?

Reply:
I would say fix typos there, but tests are green so far so its better to ship it 😅